packages/www/components/WebhookDetails/index.tsx

```diff
@@ -26,11 +29,39 @@ import {
 import { useApi } from "hooks";
 import DeleteDialog from "../WebhookDialogs/DeleteDialog";
 import LogsContainer from "./LogsContainer";
+import { Webhook } from "@livepeer.studio/api";
 
-const StyledPencil = styled(Pencil1Icon, {
-  mr: "$1",
-  width: 12,
-  height: 12,
+const eventOptions: Webhook["events"] = [
+  "playback.accessControl",
+  "stream.started",
+  "stream.idle",
+  "recording.ready",
+  "recording.started",
+  "recording.waiting",
+  "multistream.connected",
+  "multistream.error",
+  "multistream.disconnected",
+  "asset.created",
+  "asset.updated",
+  "asset.ready",
+  "asset.failed",
+  "asset.deleted",
+  "task.spawned",
+  "task.updated",
+  "task.completed",
+  "task.failed",
+];
+
+const DateInput = styled("input", {
```

Comment:
filtering by date does not work for me in the preview. I always only see "Waiting for logs...". Could you check?

<img width="821" alt="image" src="https://github.com/livepeer/studio/assets/2834997/d506f97e-e718-405a-9f56-48db7c6a719c">